.java-version

```diff
@@ -0,0 +1 @@
+17
```

Comment:
This file is missing the final newline.

Reply:
Thanks, fixed in https://github.com/package-url/packageurl-java/pull/178/commits/da461439c37e610491c418117155a832af6bfc74.